AdminWebsite/AdminWebsite.Services.Generator/booking-api-csharp.nswag

```diff
@@ -82,9 +82,9 @@
       "typeNameGeneratorType": null,
       "propertyNameGeneratorType": null,
       "enumNameGeneratorType": null,
-      "serviceHost": "http://localhost:5301",
+      "serviceHost": "http://localhost:5300",
       "serviceSchemes": null,
-      "output": "../AdminWebsite.BookingsAPI.Client/BookingsApiClient.cs"
+      "output": "../VideoWeb.Services/BookingsApiClient.cs"
```

Comment:
VideoWeb ?

Reply:
ah was a copy from video-web project when the config got corrupted. ill revert this now